### PR TITLE
fixed knifes not embedding right

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
fix: combat knifes embedding actually works now
balance: chances of embedding is 30%
/:cl:
trow_speed makes it work right

### CODE REVIEW

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
Atomize Fix and Balance changes OR document all changes in PR desc and have the title properly reflect the PR (it's not just a fix).

**Comment by XDTM** [comment]
isn't this just a nerf? what was the issue?

**Comment by ExcessiveUseOfCobblestone** [comment]
throwspeed was buffed from 3 to 4

**Comment by Tlaltecuhtli** [comment]
it works on local

File: code/game/objects/items/kitchen.dm [hunk 1]
```diff
@@ -141,7 +142,7 @@
 	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
 	desc = "A sharpened bone. The bare minimum in survival."
-	embedding = list("embedded_pain_multiplier" = 4, "embed_chance" = 35, "embedded_fall_chance" = 10)
+	embedding = list("embedded_pain_multiplier" = 2, "embed_chance" = 20, "embedded_fall_chance" = 10)
```
Comment (ShizCalev) [inline code/game/objects/items/kitchen.dm]: was this meant to be 30 to match the others?

Comment (Tlaltecuhtli) [reply code/game/objects/items/kitchen.dm]: its a piece of bone it should be less strong

**Comment by PKPenguin321** [comment]
horrible commit history

File: code/game/objects/items/kitchen.dm [hunk 2]
```diff
@@ -119,16 +119,17 @@
 	name = "combat knife"
 	icon_state = "buckknife"
 	desc = "A military combat utility survival knife."
-	embedding = list("embedded_pain_multiplier" = 4, "embed_chance" = 65, "embedded_fall_chance" = 10)
+	embedding = list("embedded_pain_multiplier" = 3, "embed_chance" = 30, "embedded_fall_chance" = 10)
 	force = 20
 	throwforce = 20
+	throw_speed = 4
```
Comment (ShizCalev) [inline code/game/objects/items/kitchen.dm]: EMBED_THROWSPEED_THRESHOLD

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/objects/items/kitchen.dm (hunk 1 above)
Comment (ShizCalev) [reply code/game/objects/items/kitchen.dm]: so it's just a nerf in a fix pr. kay.

**Comment by ShizCalev** [comment]
Make a changelog, please.

**Comment by optimumtact** [comment]
garbage PR for all the reasons enumerated above

**Pull request closed without merging**